content/arduino-cloud/02.features/10.triggers/triggers.md

```diff
@@ -0,0 +1,120 @@
+
+**3.** Go to [Arduino Cloud home](https://cloud.arduino.cc/home/) and click on **Triggers**.
+ 
+
+![Triggers in Homepage](./assets/triggerHomepage.png)
+
+**4.** Click **Add Trigger**
+
+![Add Trigger](./assets/addTrigger.png)
+
+**5.** Click **Cloud Variable**
+
+![Select Variable](./assets/selectVariable.png)
+
+ then select your **Thing**, your **Variable**, and finally press on **Link Variable**.
+
+![Link Variable](./assets/linkVariable.png)
+
+**6.** Next, select your **Action**, in our case **Email**. A new window will pop up in which we can create our personalized message, which is sent each time the trigger is activated.
+
+![Select Action](./assets/selectAction.png)
+
+You can even include **dynamic data** such as `{variable.timestamp}` or `{variable.value}` by using the tags shown at the bottom.
+
+![Email Form with Dynamic Tags](./assets/emailForm.png)
+
+**7.** Once you're finished press **Done**. The final step is to turn on the **State** switch and now your trigger is ready to be used.
+
+![Activate Trigger](./assets/activateTrigger.png)
+
+**8.** Try it out! Press the button and within a couple of seconds, you should receive an email telling you that the trigger has been activated showing you the message that you create in the previous step.
+
+## Example Code
+
+```arduino
+#include "thingProperties.h"
+
+const int btnPin = 3;
+
+int buttonState = 0;
+
+void setup() {
+  // Initialize serial and wait for port to open:
+  Serial.begin(9600);
+  // This delay gives the chance to wait for a Serial Monitor without blocking if none is found
+  delay(1500); 
+
+  // Defined in thingProperties.h
+  initProperties();
+
+  // Connect to Arduino IoT Cloud
+  ArduinoCloud.begin(ArduinoIoTPreferredConnection, false, "mqtts-sa.iot.oniudra.cc");
+  
+  /*
+     The following function allows you to obtain more information
+     related to the state of network and IoT Cloud connection and errors
+     the higher number the more granular information you’ll get.
+     The default is 0 (only errors).
+     Maximum is 4
+ */
+  setDebugMessageLevel(2);
+  ArduinoCloud.printDebugInfo();
+  
+  pinMode(btnPin, INPUT);
+  pinMode(LED_BUILTIN, OUTPUT);
+}
+
+void loop() {
+  ArduinoCloud.update();
+  
+  buttonState = digitalRead(btnPin);
+  
+  if (buttonState == HIGH) {
+    digitalWrite(LED_BUILTIN, LOW);
+    button = true;
+  } else {
+    digitalWrite(LED_BUILTIN, HIGH);
+    button = false;
+  }
+}
+```
+
+## Summary
+Triggers are a simple but extremely powerful feature allowing you to stay on track with all your Arduino IoT Cloud projects. Instead of constantly checking your dashboards to see the newest update just create a trigger notifying you as soon as changes occur. In this tutorial, we used just **one trigger** in combination with a simple demo sketch but you can create **as many Triggers as you need** to automate your projects.
```

Comment:
Maybe we can list out a couple of interesting use cases here as well, like
- PIR sensor making a simple house intrusion alert system
- Plant needs watering
- and so on